apps/content-gateway-loader/src/app/app.ts

```diff
@@ -24,53 +24,43 @@ import { withMessage } from "io-ts-types";
 import { join } from "path";
 import { createJobRepository } from "../repository/PrismaJobRepository";
 
-export const createApp = async (prisma: PrismaClient) => {
-    const CGA_URL =
-        process.env.CGA_URL || programError("You must specify CGA_URL");
-    const YOUTUBE_API_KEY =
-        process.env.YOUTUBE_API_KEY ||
-        programError("You must specify YOUTUBE_API_KEY");
-    const GHOST_API_KEY =
-        process.env.GHOST_API_KEY ||
-        programError("You must specify GHOST_API_KEY");
-    const SNAPSHOT_SPACES =
-        process.env.SNAPSHOT_SPACES?.split(",") ||
-        programError("You must specify SNAPSHOT_SPACES");
-    const DISCORD_BOT_TOKEN =
-        process.env.DISCORD_BOT_TOKEN ||
-        programError("You must specify DISCORD_BOT_TOKEN");
-    const DISCORD_CHANNEL =
-        process.env.DISCORD_CHANNEL ||
-        programError("You must specify GHOST_API_KEY");
-    const env =
-        process.env.NODE_ENV ?? programError("You must specify NODE_ENV");
+export type AppParams = {
+    nodeEnv: string;
+    resetDb: boolean;
+    addFrontend: boolean;
+    prisma: PrismaClient;
+    cgaAPIKey: string;
+    cgaURL: string;
+    youtubeAPIKey: string;
+    ghostAPIKey: string;
+    snapshotSpaces: string[];
+};
 
-    const isProd = env === "production";
-    const resetDb = process.env.RESET_DB === "true";
+export const createApp = async (appParams: AppParams) => {
+    const { prisma, nodeEnv } = appParams;
+    const isProd = nodeEnv === "production";
     const logger = createLogger("ContentGatewayLoaderApp");
-    const addFrontend = process.env.ADD_FRONTEND === "true";
 
-    if (resetDb) {
+    if (appParams.resetDb) {
         logger.info("Database reset requested. Resetting...");
         await prisma.jobLog.deleteMany({});
         await prisma.jobSchedule.deleteMany({});
     }
 
-    logger.info(`Running in ${env} mode`);
+    logger.info(`Running in ${nodeEnv} mode`);
 
     const loaderRegistry = createLoaderRegistry({
-        ghostApiKey: GHOST_API_KEY,
-        youtubeApiKey: YOUTUBE_API_KEY,
-        snapshotSpaces: SNAPSHOT_SPACES,
-        discordBotToken : DISCORD_BOT_TOKEN,
```

Comment:
discord vars missing here?

Reply:
Which ones are missing? I just copied whatever that was there. 👀